clash-protocols/src/Protocols/Df.hs

```diff
@@ -224,6 +219,14 @@ toData :: Bool -> a -> Data a
 toData False _ = NoData
 toData True a = Data a
 
+{- |
+  If the Data is `NoData`, it returns the gi value; otherwise,
+  it returns the value contained in the Data.
```

Comment:
```suggestion
  If the "Data" is 'NoData', it returns the given value; otherwise,
  it returns the value contained in the 'Data'.
```

" -> type
' -> term

AFAIK

Reply:
```suggestion
  If the t'Data' is 'NoData', it returns the given value; otherwise,
  it returns the value contained in the v'Data'.
```

Don't you mean
```
t'Type'
v'Term'
```
? https://haskell-haddock.readthedocs.io/latest/markup.html#hyperlinked-identifiers

Note you need GHC 9.0 or newer to get a Haddock that understands it, but this package already requires that.